src/views/CreateEvent.vue

```diff
@@ -1,14 +1,141 @@
 <template>
   <div class="container">
     <h1>This is Create Event page</h1>
+    <form>        
+        <div class="input__wrapper">
+          <label>Event ID: </label>
+          <input type="text" v-model="item.id">
```

Comment:
You don't need specify ID manually, just generate a random number and put it to the new event payload